app/src/routes/v3/teams/teamUsers.router.ts

```diff
@@ -108,12 +99,7 @@ router.patch(
   isAdminOrManager,
   async ctx => {
     const { teamUserId } = ctx.params;
-    const { body } = <TRequest>ctx.request;
-
-    if (body.role === EUserRole.Administrator) {
```

Comment:
removed this part because `validatorMiddleware(updateTeamUsersInput)` will throw a validation error if `body.role === "administrator"`, so no need to check it again here. Typescript called me out of this!